scripts/quarantine.yaml

```diff
@@ -6,10 +6,3 @@
 #    - None
 #  platforms:
 #    - all
-
-- scenarios:
-    - applications.nrf_desktop.zrelease
-  platforms:
-    - nrf52833dk_nrf52820
-    - nrf52820dongle_nrf52820
-  comment: "Temporary disabled due to flash overflow"
```

Comment:
emtpty quarantine causes errors in twister. To have it "empty", please copy and uncomment the above entry